resources/sass/components/_labels.scss

```diff
@@ -4,29 +4,42 @@
     border: 0;
     border-radius: $b-radius-secondary;
     color: $c-gray2;
+    cursor: pointer;
     display: inline-flex;
-    font-family: $ff-open-sans;
+    font-family: $ff-helvetica-neue;
     font-size: $fs-l;
-    font-weight: 600;
+    font-weight: 400;
     justify-content: center;
     line-height: $lh-l;
-    min-width: 136px;
     outline: none;
-    padding-bottom: $indent-s;
-    padding-top: $indent-s;
+    padding: $indent-s $indent-m;
 
     &::after {
         color: $c-gray2;
         content: 'J';
```

Comment:
better to use separate element for icons, for example <span class="..."></span> and not redefine icon on the whole component with :after or :before